sql/sql_select.cc

```diff
@@ -673,7 +673,7 @@ setup_without_group(THD *thd, Ref_ptr_array ref_pointer_array,
 }
 
 static int
-setup_for_system_time(THD *thd, TABLE_LIST *tables, COND **where_expr, SELECT_LEX *select_lex)
+setup_for_system_time(THD *thd, TABLE_LIST *tables, COND **where_expr, SELECT_LEX *slex)
```

Comment:
`select_lex` is a better name, IMO. There is no well-known `slex`. By well-known I mean `i`, `j`, `k`, `buf`, etc.